symfony/client/src/orangehrmAdminPlugin/components/EmployeeDropdown.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <oxd-input-field
+    type="dropdown"
+    label="Employee Name"
+    :create-options="loadEmployees"
+    :lazyLoad="true"
+  />
+</template>
+
+<script>
+import {APIService} from '@orangehrm/core/util/services/api.service';
+export default {
+  name: 'employee-dropdown',
+  setup() {
+    const http = new APIService(
+      window.appGlobal.baseUrl,
```

Comment:
Can't we move `window.appGlobal.baseUrl` into APIService? since base URL not changing